docs/system_overview/configuration.md

```diff
@@ -69,6 +81,28 @@ This file defines the behavior of the server Agent that answers individual reque
         // The amount of time a temporary password remains valid.
         "maximum_temporary_password_lifetime_in_seconds": 1000,
 
+        // (Optional)
+        // The number of seconds between runs of the CRON task which runs the delay server migration.
+        "migrate_delay_server_sleep_time_in_seconds": 5,
+
+        // (Optional)
+        // Sets the timeout for receive operations via ZeroMQ's setsockopt API for delay server migration communications.
+        // The following values are considered for controlling zmq_recv:
+        //  - 0:    Returns immediately, with a EAGAIN error if there is no message to receive.
+        //  - -1:   Blocks until a message is available.
+        //  - Else: Waits for a message for that amount of time in milliseconds before returning with an EAGAIN error.
+        // For more details, search for ZMQ_RCVTIMEO here: http://api.zeromq.org/master:zmq-setsockopt
```

Comment:
Add an empty line after L89 and L93 to match formatting of other properties.

Align list with other text to match formatting of other properties.

Reply:
Will do